app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -16,60 +21,159 @@
 
 import com.google.android.material.textfield.TextInputEditText;
 
+import java.util.regex.Pattern;
+
 /**
-* A simple {@link WolmoFragment} subclass.
-*/
+ * A simple {@link WolmoFragment} subclass.
+ */
 public class LoginFragment extends WolmoFragment implements ILoginView {
-    LoginPresenter presenter;
+    LoginPresenter presenter = new LoginPresenter();
+
+    @BindView(R.id.vLoginButton)
+    Button vLoginButton;
```

Comment:
Move this line together with the BindView declaration. The same for the other views

Reply:
What happened with this?